upload.prx.org/upload.prx.org.yml

```diff
@@ -83,3 +113,22 @@ Resources:
       Role: !GetAtt LambdaBasicExecutionIAMRole.Arn
       Runtime: nodejs4.3
```

Comment:
Might as well try to use node6 since it's available and probably requires no changes